lib/rules/block-indentation.js

```diff
@@ -82,6 +82,16 @@ function isControlChar(char) {
 }
 
 module.exports = class BlockIndentation extends Rule {
+  static get meta() {
```

Comment:
any reason for this being a property method instead of something like `static meta = { ... }`?

Reply:
I don't think static class fields are supported on our supported Node versions. Could be wrong though...